formatTest/typeCheckedTests/expected_output/knownMlIssues.re

```diff
@@ -12,12 +11,11 @@ let blahCurriedX x =>
   fun
   | Red 10
   | Black 20
-  | Green 10 => 1
-  /* After or pattern green */
+  | Green 10 =>
+    1 /* After or pattern green */
```

Comment:
It should. We don't have the loc information for the case so the algorithm doesn't know where to attach to. 

The algorithm does try to attach the comment to the entire case group (from |RED to 1), but figures out that the entire group is not on the same line. 
```
  | Red 10
  | Black 20
  | Green 10 -> 1 (* After or pattern green *)
```